Gemfile.lock

```diff
@@ -2,6 +2,7 @@ GEM
   remote: https://rubygems.org/
```

Comment:
I forget the consensus on adding the Gemfile.lock into the repo but it looks like it should be according to [StackOverflow](https://stackoverflow.com/a/4151540) (i.e. strangers on the Internet)

Reply:
yeah, not sure while the Gemfile got pushed this time for some reason